packages/in_app_purchase/in_app_purchase_android/android/src/test/java/io/flutter/plugins/inapppurchase/TranslatorTest.java

```diff
@@ -164,20 +163,20 @@ public void fromBillingResult() {
             .setDebugMessage("dummy debug message")
             .setResponseCode(BillingClient.BillingResponseCode.OK)
             .build();
-    Map<String, Object> billingResultMap = Translator.fromBillingResult(newBillingResult);
+    Messages.PlatformBillingResult platformResult = Translator.fromBillingResult(newBillingResult);
 
-    assertEquals(billingResultMap.get("responseCode"), newBillingResult.getResponseCode());
-    assertEquals(billingResultMap.get("debugMessage"), newBillingResult.getDebugMessage());
+    assertEquals(platformResult.getResponseCode().longValue(), newBillingResult.getResponseCode());
+    assertEquals(platformResult.getDebugMessage(), newBillingResult.getDebugMessage());
   }
 
   @Test
   public void fromBillingResult_debugMessageNull() {
     BillingResult newBillingResult =
         BillingResult.newBuilder().setResponseCode(BillingClient.BillingResponseCode.OK).build();
-    Map<String, Object> billingResultMap = Translator.fromBillingResult(newBillingResult);
+    Messages.PlatformBillingResult platformResult = Translator.fromBillingResult(newBillingResult);
 
-    assertEquals(billingResultMap.get("responseCode"), newBillingResult.getResponseCode());
-    assertEquals(billingResultMap.get("debugMessage"), newBillingResult.getDebugMessage());
+    assertEquals(platformResult.getResponseCode().longValue(), newBillingResult.getResponseCode());
```

Comment:
It seems odd that we would need to get a long value here are you sure  it is needed?

Reply:
Where I have these in tests it's because the Java compiler was angry at me for trying to pass a `Long` and a `long` (or sometimes `int`) to `assertEquals`, which wants the same kind of object.